scripts/github/release-artifacts.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+help() {
+  cat <<EOF
+Usage: $(basename "$0") [COMMAND] [OPTIONS]
+
+Options:
+  -h, --help                        Display this text.
+  --tag           <tag>             The release tag.
+  --workflow      <workflow>        The workflow which builds/archives the artifacts.
+  --repo-org      <repo-org>        The repo's owner organization.
+  --upload        <repos>           Upload artifacts to the given repos.
+
+Command:
+  download                          Download the artifacts.
+  upload                            Upload the artifacts.
+
+Examples:
+  $(basename "$0") --release-tag v3.0.0
+EOF
+}
+
+echo_stderr() {
+  echo -e "${1}" >&2
+}
+
+die() {
+  local _return="${2:-1}"
+  echo_stderr "$1"
+  exit "${_return}"
+}
```

Comment:
If you want, you can use the functions from here: https://github.com/openebs/mayastor-extensions/blob/develop/scripts/utils/log.sh

The echo_stderr is log_to_stderr
The die is log_fatal